.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: npm install
+    command: 'echo "TODO: Replace with command to start project"'
```

Comment:
This is not a pull request but an option request, should be placed under issues instead.